app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/returns/acquisitiondetails/acquisition_fees.scala.html

```diff
@@ -39,7 +39,7 @@
     backLinkComponent: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.back_link,
     returnToSummaryLink: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.return_to_summary_link,
     unitInput: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.unit_input,
-    caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption,
+    caption: uk.gov.hmrc.cgtpropertydisposalsfrontend.views.html.components.caption_govuk,
```

Comment:
I think, we need to use different approach. you can fix it using PageHeading component like 
`label = pageHeading(title, Some(messages("returns.disposal-details.caption"))),`

try this one if works okay. otherwise there is another approach too.